### DISCUSSION TITLE
Revisit styling for standalone pages

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/methods#94] [Ideas]
@Bixal/methods, playing around with some highlighting for utility nav and light visual tweak. Open to other ideas as well. 

![image](https://user-images.githubusercontent.com/5177337/107091757-47c6c700-67d0-11eb-91a1-fa30cb2e34b6.png)

_Originally posted by @pglevy in https://github.com/Bixal/methods/issues/85#issuecomment-774302002_

**Comment by katharine-carr** [comment]
Hey @pglevy what do you think about something like this?
![Screen Shot 2021-02-18 at 2 01 36 PM](https://user-images.githubusercontent.com/50249974/108407626-fbc54a80-71f1-11eb-86df-133e362be03c.png)

**Reply by pglevy** [reply]
My initial impression is the suggestion button has a bit of a "disabled" vibe to it. 🤔 I think the capitalization solves some issues though.

**Reply by katharine-carr** [reply]
Oh! Good point. Maybe there is another color we can try that is unobtrusive. FYI i also set the utility nav text to one pixel size down (14 instead of 15). Up to you if that feels too small.
